README.md

```diff
@@ -12,34 +12,32 @@ Bozza ver. 01.047 – 2020-04-12 &ndash; Scritto con [***Mark Text***](https://m
-  Spendere 5 o 10 minuti del vostro tempo per aprire un acount gratuito su Diaspora sarà l'occasione di uscire un po' dalle grinfie di Facebook, con la sua censura e la sua connivenza con *Big Data*.
+  La versione che più vi è piaciuta potrete anche copiarla (e lo stesso dicasi per qualunque altra).  
+  **Se disponete di account, nick o una qualunque "facciata web" potrete segnalare l'esitenza del manifesto tramite la stessa.**  
+  Vi ringrazieremo se spenderete 5-10 minuti del vostro tempo per aprire un account gratuito su Diaspora e sarà anche l'occasione di uscire un po' dalle grinfie di Facebook, con la sua censura e la sua connivenza con *Big Data*.
 
 &nbsp;
 
-Almeno durante un primo periodo, questo sarà un lavoro in evoluzione, anche se vorremo marcare il raggiungimento di certe tappe di sviluppo del pensiero e dell'azione (potremmo valutare se effettuare delle *release*).
+Almeno per un primo periodo questo manifesto é da cosiderare come un lavoro in evoluzione, anche se vorremmo marcare il raggiungimento di certe tappe di sviluppo del pensiero e dell'azione entro breve tempo (potremmo valutare se effettuare delle *release*).
 
 **SE AVETE DELLE IDEE CON CUI CONTRIBUIRE, FATEVI AVANTI.**
 
-Abbiamo per ora una sola regola: **niente aggressioni**, **niente stroncature di nessun apporto**. Solo così la creatività di tutti potrà dare i frutti migliori.
+Abbiamo, per ora, una sola chiara regola: **niente aggressioni**, **niente stroncature o commenti di nessun apporto**, affinché la creatività di tutti potrà portare ai frutti migliori.
 
-Saranno benvenuti **giuristi**, **costituzionalisti**, **economisti**, **filosofi**, **scienziati**, **informatici**, **matematici**, **giornalisti**, ***bloggers*** e **pensatori e intellettuali in genere**.
+Saranno benvenuti particolarmente categorie di **giuristi**, **costituzionalisti**, **economisti**, **filosofi**, **scienziati**, **informatici**, **matematici**, **giornalisti**, ***bloggers*** e **pensatori e intellettuali**, così come **traduttori** e **artisti** in genere.
 
-Saranno anche benvenuti **traduttori** e **artisti**.
-
-Potremmo valutare l'aggiunta di note in calce con *ulteriori* riferimenti in merito a vari argomenti toccati da questo documento. Non l'abbiamo per ora considerato prioritario con rispetto all'avere una prima stesura e a lavorare per diffonderla senza ulteriori indugi. In quest'ambito non pretendiamo di dimostrare niente, **stiamo *constatando*, preoccupati, e proponendo una bandiera fatta di ideali in cui crediamo**.
+Valuteremo l'aggiunta di note in calce con *ulteriori* riferimenti ed approfondimenti in merito ai vari argomenti toccati in questo manifesto. Non l'abbiamo ancora elaborato in rapporto all'esigenza prioritaria di pubblicare una prima stesura, nonché lavorare per diffonderla senza ulteriori indugi. In quest'ambito non pretendiamo di dimostrare nulla, **stiamo *constatando*, preoccupati, e proponendo una bandiera fatta di ideali in cui crediamo**.
```

Comment:
"toccati in" perfetto

"elaborato in rapporto" mi pare non trasmetta l'idea di subordinazione di un'esigenza rispetto all'altra

Cerco di raffinare entrambe togliendo rumore:

Valuteremo l'aggiunta di note in calce con *ulteriori* riferimenti ed approfondimenti in merito ai vari argomenti toccati in questo manifesto. Abbiamo per ora considerato *prioritaria* l'esigenza di pubblicare *una prima stesura* e di lavorare per *diffonderla* senza ulteriori indugi. In quest'ambito non pretendiamo di dimostrare nulla, **stiamo *constatando*, preoccupati, e proponendo una bandiera fatta di ideali in cui crediamo**.

(Ho anche messo in corsivo alcune parole chiave.)

Reply:
idem, non commenti, modifichi pull request o preferisci che modifichi io dalla mia copia locale e faccia push?